src/ServiceControl.AcceptanceTests/AcceptanceTest.cs

```diff
@@ -87,7 +87,7 @@ protected AcceptanceTest()
         [SetUp]
         public void Setup()
         {
-            port = FindAvailablePort(33333);
+            port = FindAvailablePort(33335);
```

Comment:
Why change the default port? Just so it doesn't conflict with your existing SC instance?

Do we know how much time we spend in this method per run? I wonder if we couldn't be smarter by building a circular buffer and always moving forwards.